profiles/management/commands/retire_users.py

```diff
@@ -0,0 +1,61 @@
+"""
+Validate course prices
+"""
+from django.contrib.auth.models import User
+from django.core.management import BaseCommand, CommandError
+from social_django.models import UserSocialAuth
+
+from dashboard.models import ProgramEnrollment
+
+
+class Command(BaseCommand):
+    """
+    Validate prices and financial aid
+    """
+    help = "Retire one or multiple users"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-u',
+            '--user',
+            action='append',
+            default=[],
+            dest='users',
+            help="Single or multiple user name"
+        ),
+
+    def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
+        user_names = kwargs.get("users", [])
+
+        if len(user_names) <= 0:
+            # show error when no user selected.
+            raise CommandError("Please select user(s)")
+
+        for user_name in user_names:
+            """retire user"""
+            if not user_name:
+                self.stdout.write("Invalid username.")
```

Comment:
Please include the username in the error message

Reply:
this is for invalid user, i.e null or empty string. Can be produce with command
`docker-compose run web ./manage.py retire_users --user=`

Reply:
But it's specifically for a username that's false, so the error could be clearer. How about "Username required"